backends/dialogs/macosx/macosx-dialogs.mm

```diff
@@ -175,25 +163,28 @@ - (IBAction) showHiddenFiles : (id) sender {
 
 
 	NSOpenPanel *panel = [NSOpenPanel openPanel];
-	[panel setCanChooseFiles:!_isDirBrowser];
-	[panel setCanChooseDirectories:_isDirBrowser];
-	if (_isDirBrowser)
+	[panel setCanChooseFiles:!isDirBrowser];
+	[panel setCanChooseDirectories:isDirBrowser];
+	if (isDirBrowser)
 		[panel setTreatsFilePackagesAsDirectories:true];
-	[panel setTitle:(NSString *)_titleRef];
-	[panel setPrompt:(NSString *)_chooseRef];
+	[panel setTitle:titleRef];
```

Comment:
This results in a compilation error. You need to keep the explicit cast to `NSString*` here (and on the next line).